### PR TITLE
fix #277869: wrong naming of rows in timeline with grand staff

### PR DESCRIPTION
**Pull request by jthistle**
Resolves: https://musescore.org/en/node/277869

### CODE REVIEW

File: mscore/timeline.h [hunk 1]
```diff
@@ -192,6 +192,8 @@ class Timeline : public QGraphicsView {
       Timeline(TDockWidget* dock_widget, QWidget* parent = 0);
       int correctPart(int stave);
 
+      QList<Part*> getParts();
```
Comment (dmitrio95) [inline mscore/timeline.h]: Better make it private perhaps? Or is it intended to be used outside `Timeline`?

Comment (jthistle) [reply mscore/timeline.h]: no, you're right, it should be private.

**Comment by jthistle** [comment]
Private now @dmitrio95

**Comment by dmitrio95** [comment]
Well, great! Other changes seem to be good in my opinion.

**Pull request merged**